operations/security_and_authentication/Recovering_from_Mismatched_BMC_Credentials.md

```diff
@@ -0,0 +1,144 @@
+# Recovering from Mismatched BMC Credentials
+
+This procedure is aimed recovering from the situation when new or replacement hardware has `root` credentials that do not match the system's current default `root` user credentials.
+
+This type of problem can occur in the following scenarios:
+
+- The Site has customized the default `root` credentials using either the
+  [Updating the Liquid-Cooled EX Cabinet CEC with Default Credentials after a CEC Password Change](Updating_the_Liquid-Cooled_EX_Cabinet_Default_Credentials_after_a_CEC_Password_Change.md) or
+  [Update Default Air-Cooled BMC and Leaf-BMC Switch SNMP Credentials](Update_Default_Air-Cooled_BMC_and_Leaf_BMC_Switch_SNMP_Credentials.md) procedures.
+- Hardware has the factory default `root` password or a different known `root` password configured. Such as a pieces of hardware that moved from a different system with customized default `root` password.
+
+## Procedure
+
+1. Specify the BMC hostname with the mismatched credentials:
+
+    ```bash
+    ncn-m001# BMC=x1000c0r1b0
+    ```
+
+1. Specify the current `root` user password for the BMC:
+
+    > Depending on the origin of the piece of hardware, this could be the factory default password or a different systems default password.
```

Comment:
```suggestion
    > Depending on the origin of the piece of hardware, this could be the factory default password or a different system's default password.
```